easybib/libraries/easybib.rb

```diff
@@ -92,18 +93,18 @@ def is_aws(node = self.node)
   #
   # @return [String]
   def get_hostname(node = self.node, fail_if_nil = false)
-    if !get_cluster_name(node).empty?
-      instance    = get_instance(node)
-      my_hostname = instance['hostname']
-    else
-      # node.json
-      my_hostname = if node['server_name']
-                      node['server_name']
-                    # from 'ohai'
-                    else
-                      node['hostname']
-                    end
+    opsworks_hostname = false
```

Comment:
Tried that initially, wont work. There is some "weird" object coming back, not just nil